pkg/cgroup/client.go

```diff
@@ -227,6 +227,11 @@ func (c *Client) findCgroupPathForContainerID(containerID string) (string, ID) {
 	retPath := ""
 	rootDir := c.getCgroupSearchBasePath()
 	var cgroupID ID
+	// TODO(anjmao): This is can be really slow and expensive. We should consider another approach:
+	// 1. During cgroup mkdir find cgroup file from /proc/<pid>/cgroup
```

Comment:
This will not work, as the PID for `cgroup mkdir` would be the one outside the container.